plugins/woocommerce/includes/abstracts/abstract-wc-product.php

```diff
@@ -2021,7 +2021,14 @@ public function add_to_cart_description() {
 	public function get_image( $size = 'woocommerce_thumbnail', $attr = array(), $placeholder = true ) {
 		$image = '';
 		if ( $this->get_image_id() ) {
-			$image = wp_get_attachment_image( $this->get_image_id(), $size, false, $attr );
+			$image_alt = get_post_meta( $this->get_image_id(), '_wp_attachment_image_alt', true );
```

Comment:
That's correct, but this approach is used in other areas as a default. Probably the WooCommerce team should decide how they'd like to handle this consistently across the project.

Reply:
Ups, sorry, I missed that comment. I'm curious about why we shouldn't use the alt text, though.

The Cart table already has a cell with the product name, what's the benefit of using the product name for the image alt attribute, then? That info is already there. :thinking:

Reply:
I do tend to agree that the alt text should not be the image alt but the product name. Because in the context of showing that image, it's meant to related to the actual product.